integration-tests/src/test/java/com/ca/mfaas/utils/http/HttpSecurityUtils.java

```diff
@@ -22,14 +22,14 @@
 import java.io.IOException;
 import java.net.URI;
 
+@UtilityClass
 public class HttpSecurityUtils {
 
     public static String getCookieForGateway() throws IOException {
-        AuthConfigurationProperties authConfigurationProperties = new AuthConfigurationProperties();
         Credentials credentials = ConfigReader.environmentConfiguration().getCredentials();
         String user = credentials.getUser();
         String password = credentials.getPassword();
-        URI uri = HttpRequestUtils.getUriFromGateway(authConfigurationProperties.getGatewayLoginEndpoint());
+        URI uri = HttpRequestUtils.getUriFromGateway("/api/v1/gateway/auth/login"); //can be centralized in config java file
```

Comment:
@ilkinabdullayev, could you please explain, why did you replace on hardcoded value?